### PR TITLE
fix: warnings in add and select tests

### PR DESCRIPTION
**Pull request by davidmenendez**
Contributes to #2732 

in addition to the aformentioned issue, i also noticed some other warning in the console related to missing labels, which i also addressed here as well as a small css issue.

### CODE REVIEW

**Comment by elycheea** [comment]
@davidmenendez updated the title for a hopefully more accurate representation than the issue number. 😜

**Comment by davidmenendez** [comment]
@elycheea thank you! fixt

File: packages/cloud-cognitive/src/components/AddSelect/AddSelectFilter.js [hunk 1]
```diff
@@ -112,15 +112,18 @@ export let AddSelectFilter = ({
...
-                  <Dropdown
-                    id={filterOpts.id}
-                    key={filterOpts.id}
-                    titleText={filterOpts.label}
-                    items={filterOpts.opts}
-                    onChange={(value) => onchangeHandler(value, filterOpts.id)}
-                    selectedItem={getSelectedItem(filterOpts.id)}
-                    label={placeholder}
-                  />
+                  <Layer key={filterOpts.id}>
```
Comment (lee-chase) [inline packages/cloud-cognitive/src/components/AddSelect/AddSelectFilter.js:115]: Does the layer need to be inside the map function?

**Review by lee-chase — COMMENTED** [review]
Ready to approve, just wondered why the Layer is inside the map.

File: packages/cloud-cognitive/src/components/AddSelect/AddSelectFilter.js (hunk 1 above)
Comment (davidmenendez) [reply packages/cloud-cognitive/src/components/AddSelect/AddSelectFilter.js:115]: my understanding about how layer works is that it needs to surround each individual dropdown https://react.carbondesignsystem.com/?path=/story/components-dropdown--with-layer

Comment (davidmenendez) [reply packages/cloud-cognitive/src/components/AddSelect/AddSelectFilter.js:115]: actually sorry i think i misunderstood the implementation. making some changes and then pushing an update

Comment (elycheea) [reply packages/cloud-cognitive/src/components/AddSelect/AddSelectFilter.js:115]: I don’t think it needs to wrap each individual dropdown. I’d consider it more like a `div` that wraps a set of dropdowns on the same “layer.”

**Comment by elycheea** [comment]
@davidmenendez Also have conflicts — probably from folder rename.

**Comment by davidmenendez** [comment]
@elycheea i moved `<Layer />` to be top level. i think that should resolve it.

**Review by elycheea — COMMENTED** [review]
I reset the layer since we were seeing Layer 3 within another Layer 3.

image

Also moved the className to the `Layer` component since we were seeing this.
image

**Review by elycheea — APPROVED** [review]

**Pull request merged**